.prettierignore

```diff
@@ -0,0 +1,5 @@
+.eslintrc.js
```

Comment:
Can be ignored since it's auto generated

Reply:
added to .gitignore